praw/models/reddit/subreddit.py

```diff
@@ -1290,74 +1290,68 @@ def submit(
         "spoiler",
     )
     def submit_gallery(
-        self,
-        title: str,
-        images: list[dict[str, str]],
-        *,
-        collection_id: str | None = None,
-        discussion_type: str | None = None,
-        flair_id: str | None = None,
-        flair_text: str | None = None,
-        nsfw: bool = False,
-        send_replies: bool = True,
-        spoiler: bool = False,
+            self,
+            title: str,
+            images: list[dict[str, str]],
+            text: str = '',
```

Comment:
This needs to be named `selftext` and sorted into the keyword arguments alphabetically.